src/commonMain/kotlin/com/configcat/DateTimeUtils.kt

```diff
@@ -3,28 +3,13 @@ package com.configcat
 import com.soywiz.klock.*
 
 internal object DateTimeUtils {
-
-    /**
-     * HTTP Date header formatter. Date: day-name, day month year hour:minute:second GMT
-     *
-     * @see <a href="https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Date">mdn docs</a>
-     */
-    private val dateFormat: DateFormat = DateFormat("EEE, dd MMM yyyy HH:mm:ss z")
-
-    fun isValidDate(dateString: String): Boolean {
+    fun isValidDate(fetchTime: String): Boolean {
         try {
-            dateFormat.parse(dateString)
-        } catch (e: DateException) {
+            val fetchTimeSeconds = fetchTime.toDouble()
```

Comment:
According to the spec, we expect a 64-bit integer here so I suggest parsing it as an integer instead of a decimal value.